bclconvert.wdl

```diff
@@ -19,189 +15,167 @@ struct FastqCollection {
 }
 
 
-
-
 workflow bclconvert {
+
   input {
     String runDirectory
-    String runName
-    Array[Sample] samples
+    String mode
+    Sample sample 
```

Comment:
```suggestion
    Array[String]+ barcodes
    String name
```

Reply:
I would prefer the original implementation

Reply:
Putting everything at the top level simplifies what we will need to do for the injection process. Why do we need a struct if there will only ever be one instance of it?